WordPress/Classes/Networking/JetpackServiceRemote.m

```diff
@@ -59,4 +60,31 @@ - (void)validateJetpackUsername:(NSString *)username
          }];
 }
 
-@end
\ No newline at end of file
+/**
+    Check if the specified site is a Jetpack site.  The success block
+    receives a bool indicating if the site hass Jetpack.
+ */
+- (void)checkSiteHasJetpack:(NSURL *)siteURL
+                    success:(void (^)(BOOL isJetpack))success
+                    failure:(void (^)(NSError *error))failure
+{
+    NSString *siteStr = [[NSString stringWithFormat:@"%@%@", siteURL.host, siteURL.path]
+                         stringByAddingPercentEncodingWithAllowedCharacters:NSCharacterSet.URLHostAllowedCharacterSet];
```

Comment:
I thought so too initially but it allows a `/` character which if included in the blog's path will cause the API call to resolve to a 404.  Try it with a self-hosted site installed to a subfolder to reproduce.  We need to escape any `\` characters in the `$address` component of the API call.  Hence using `.URLHostAllowedCharacterSet`.